scripts/build-canister.sh

```diff
@@ -34,6 +34,14 @@ if [[ "$STATUS" -eq "0" ]]; then
     ./target/bin/ic-wasm \
     "./target/$TARGET/release/$CANISTER.wasm" \
     -o "./target/$TARGET/release/$CANISTER.wasm" shrink
+
+    if [[ "$CANISTER" == "ic-btc-canister" ]]; then
+    	./target/bin/ic-wasm \
+    	"./target/$TARGET/release/$CANISTER.wasm" \
+    	-o "./target/$TARGET/release/$CANISTER.wasm" \
+    	metadata candid:service -f "$SCRIPT_DIR/../canister/candid.did" -v public
```

Comment:
Is there anything we can run locally to see the metadata? I wonder how one can verify that this command works.

Reply:
Oh yeah, I forgot to add this. If you build the canisters using docker; extract the bitcoin canister's Wasm, then unzip it and run `ic-wasm <bitcoin_canister_wasm> info` you can see that there's a public metadata section called `candid:service`.

Reply:
I added a workflow to check this which is even better I think.